osu.Game/Screens/Play/PauseContainer.cs

```diff
@@ -122,6 +120,9 @@ protected override void Update()
             if (!game.IsActive && CanPause)
                 Pause();
 
+            if (!IsPaused)
+                framedClock.ProcessFrame();

```

Comment:
I don't like that we're processing what looks like the gameplay clock(?) in the pause container. I would rather that this was done at a `Player` level.

Reply:
Hmm, the logic here was that only things which are children of the pause container use this clock. I was hoping eventually we wouldn't need to pass it to anything but the pause container.

The `PauseContainer` was supposed to move all pause/go logic out of `Player` itself